ManualDeploymentExtended.md

```diff
@@ -0,0 +1,966 @@
+# Extended Manual Setup
+
+This document lists steps for manually deploying BTCPay Server and additional related components. Following these steps is likely to take a long time. A shorter and more pragmatic approach is to use a [docker based deployment](https://github.com/btcpayserver/btcpayserver-docker).
+
+#### Warning: Not recommended for production use
+
+Manual installation is NOT recommended for production use unless you are very confident with your Operating System and Bitcoin security expertise. If you are unsure use [docker](https://github.com/btcpayserver/btcpayserver-docker).
+
+## Installation Steps Overview
+
+The instructions in this article have been tested on Ubuntu 18.04. They should be applicable to other Linux based distributions. They are also based on all components being on the same host or virtual machine. It is possible to split the components across different hosts but these instructions don't describe that.
+
+### Prerequisites:
+
+1. Postgresql,
+2. Tor,
+3. NGINX and Let's Encrypt.
+
+### Application Components
+
+1. Bitcoin Daemon<sup>1</sup>,
+2. NBXplorer<sup>1,2</sup>,
+3. BTCPay Server<sup>1,2</sup>,
+4. Lightning Network Daemon (lnd)<sup>2</sup>,
+5. Ride The Lightning (RTL)<sup>2</sup>.
+
+<sup>1</sup> The bare minimum install of a BTCPay server only requires the starred items. Using a bare minimum configuration reduces the functionality: no lightning payments, no auto-renewal of TLS certificates, less reliable data store, less capable of handling NAT and more.
```

Comment:
There are no stars, perhaps change to "The bare minimum install of a BTCPay server only requires these items."